pkg/annotations.go

```diff
@@ -139,6 +165,34 @@ func (cronitorParser CronitorConfigParser) GetSpecifiedCronitorName() string {
 	return ""
 }
 
+// GetCronitorName returns the name to be used by Cronitor monitor.
+// Allows the namespace to be prepended or not, and allows arbitrary strings as a prefix.
+// Defaults to prepending the namespace if no pre-specified monitor name is provided by the user.
+func (cronitorParser CronitorConfigParser) GetCronitorName() string {
+	// Default behavior
+	prefix := "namespace"
```

Comment:
I like that this keeps the naming the same for anyone who is upgrading to this release. On the other hand, after our chat last week I was pretty convinced that it would be better to exclude the namespace by default. Did you land on this for compatibility sake, or are there other reasons to consider for keeping the default as `namespace`?

Reply:
I kept the current behavior so that we can gradually roll out the new behavior. I definitely think that the "new" way is more intuitive, but it'd be a pretty jarring change for folks to see during an upgrade 😅

Reply:
Sounds good!